packages/pluggableWidgets/slider-web/src/components/__tests__/Slider.spec.tsx

```diff
@@ -0,0 +1,125 @@
+import { createElement } from "react";
+import { render, screen, cleanup, fireEvent } from "@testing-library/react";
+import { Slider, SliderProps } from "../Slider";
+import { mount } from "enzyme";
+
+describe("Slider", () => {
+    afterEach(cleanup);
+
+    const defaultSliderProps = Object.freeze<SliderProps>({
+        min: -100,
+        max: 100,
+        step: 10
+    });
+
+    it("should render horizontal Slider correctly", () => {
```

Comment:
Better change the description to "renders horizontal Slider correctly" as you can read as "It" blabla..